scylla/providers/proxy_list_provider.py

```diff
@@ -35,14 +35,14 @@ def parse(self, document: PyQuery) -> [ProxyIP]:
     def urls(self) -> [str]:
         ret = []
         first_url = 'http://proxy-list.org/english/index.php?p=1'
-        sub = first_url[0:first_url.rfind('/')] # http://proxy-list.org/english
+        # sub = first_url[0:first_url.rfind('/')] # http://proxy-list.org/english
```

Comment:
Since of commenting out of this line, could we just remove it since we have history?